removestar/__main__.py

```diff
@@ -15,18 +15,16 @@
 """
 import argparse
 import glob
+import importlib
```

Comment:
Believe this needs to be `import importlib.util` on [later versions of Python](https://discuss.python.org/t/python3-11-importlib-no-longer-exposes-util/25641).

```console
    and importlib.util.find_spec("nbconvert") is not None
AttributeError: module 'importlib' has no attribute 'util'
```